build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+if command -v dpkg-query -l zip
+then
+  git clone https://github.com/AltaPay/sdk-php.git
```

Comment:
This will fetch the latest master, it would be best to specify a version
```suggestion
  git clone --branch 3.0.0 https://github.com/AltaPay/sdk-php.git
```